tr/news/_posts/2018-12-25-ruby-2-6-0-released.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: news_post
+title: "Ruby 2.6.0 Yayınlandı"
+author: "naruse"
+translator: "İsmail Arılık"
+date: 2018-12-25 00:00:00 +0000
+lang: tr
+---
+
+Ruby 2.6.0'ın yayınını duyurmaktan memnuniyet duyarız.
+
+Bu yayın birkaç yeni özellik ve performans artışı getirmektedir, en önemlileri:
+
+ * Yeni bir JIT derleyicisi.
+ * `RubyVM::AbstractSyntaxTree` modülü.
+
+## JIT [Deneysel]
+
+Ruby 2.6 bir JIT (Just-In-Time) derleyicisinin bir ilk gerçeklemesini getirmektedir.
```

Comment:
Bu cümleyi anlamadım. :(

Reply:
Doğrudan çeviri yapınca böyle oldu. Orijinal cümle şu:

> Ruby 2.6 introduces an initial implementation of a JIT (Just-In-Time) compiler.

Şöyle olabilir mi:

> Ruby 2.6 bir JIT (Just-In-Time) derleyicisinin ilk gerçeklemesini barındırmaktadır.

Reply:
`Ruby 2.6 bir JIT (Just-In-Time) derleyicisinin ilk tanımlamasını barındırmaktadır.` `implementation` burada aslında gerçekleştirmekten ziyade bir tanımla ya da uygulamayı anlatmaya çalışıyor. İngilizcesini okuyunca ne alıyoruz; Ruby 2.6'da JIT için ilk tanımlamaya giriş yapmışlar. Buna yakın bir çeviri bence daha doğru.

Reply:
3186ef2